worlds/messenger/__init__.py

```diff
@@ -130,14 +131,16 @@ def create_items(self) -> None:
             itempool += seals
 
         remaining_fill = len(self.multiworld.get_unfilled_locations(self.player)) - len(itempool)
-        filler_pool = dict(list(FILLER.items())[2:]) if remaining_fill < 10 else FILLER
-        itempool += [self.create_item(filler_item)
-                     for filler_item in
-                     self.random.choices(
-                         list(filler_pool),
-                         weights=list(filler_pool.values()),
-                         k=remaining_fill
-                     )]
+        if remaining_fill < 10:
+            filler_pool = dict(list(FILLER.items())[2:])
+            self._filler_items = [name
+                                  for name in
+                                  self.random.choices(
+                                      list(filler_pool),
+                                      weights=list(filler_pool.values()),
+                                      k=remaining_fill
+                                  )]
```

Comment:
```suggestion
            self._filler_items =  self.random.choices(
                                      list(filler_pool),
                                      weights=list(filler_pool.values()),
                                      k=remaining_fill
                                  )
```

formatting is not easily fixable in this editor.